docs/release/3.4/release-3-4-28.md

```diff
@@ -62,16 +62,18 @@ Detailed changes are below. All of the documentation can be found [here](/index.
 
 1. Upgrades from <= 3.4.0 may require merging `/etc/condor/config.d/*.rpmnew` files and a restart of HTCondor.
 
-1. GlideinWMS >= 3.4.5 uses shared port, requiring only port 9618.
+1. GlideinWMS >= 3.4.5 can use shared port. If you do so, it requires only port 9618.
    To ease the transition to shared port, the User Collector secondary collectors and CCBs support both shared and
    separate, individual ports.
-   To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in
-   `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:
 
+   -  The HTCondor configuration should be changed as suggested in the new HTCondor config files included in the RPMs, to allow the support of shared port: check the `.rpmnew` files or apply the same changes if you configure your daemons manually.
+   -  Port 9818 should be open also on standalone schedds (that earlier used only 9615).
+   -  To start using shared port, change the secondary collectors lines and the CCBs lines (if any) in `/etc/gwms-frontend/frontend.xml`, changing the address to include the shared port sinful string:
+      ```
```

Comment:
Triple backticks don't work within lists. Instead, you need to indent code blocks by 4 spaces per list level (https://opensciencegrid.org/technology/documentation/style-guide/#lists)

```suggestion

```